app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguagePage.kt

```diff
@@ -18,24 +18,22 @@ import androidx.compose.ui.unit.dp
 import com.example.awesomelanguagelearning.R
 import com.example.awesomelanguagelearning.chooseLanguage.domain.entity.ChooseLanguageListItem
 import com.example.awesomelanguagelearning.chooseLanguage.domain.entity.ChooseLanguagePageState
+import com.example.awesomelanguagelearning.chooseLanguage.ui.ChooseLanguageEvent
 import com.example.awesomelanguagelearning.chooseLanguage.ui.utils.ChooseLanguagePageParameterProvider
 import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
 import com.example.awesomelanguagelearning.core.ui.views.HorizontalSpacer
 import com.example.awesomelanguagelearning.core.ui.views.TextButton
 import com.example.awesomelanguagelearning.core.ui.views.TextTitle
-import java.util.UUID
 
 @Composable
 fun ChooseLanguagePage(
     state: ChooseLanguagePageState,
-    onNextClick: () -> Unit = {},
-    onItemClick: (UUID) -> Unit = {}
+    onEvent: (ChooseLanguageEvent) -> Unit = {}
 ) {
```

Comment:
@igel911 and here 👀

Reply:
@slothdeboss Are those modifiers in the room with us right now?